src/main/java/org/checkerframework/specimin/PrunerVisitor.java

```diff
@@ -37,45 +36,18 @@
 import com.github.javaparser.resolution.types.ResolvedType;
 import java.util.HashSet;
 import java.util.Iterator;
-import java.util.List;
+import java.util.Map;
 import java.util.NoSuchElementException;
 import java.util.Set;
 import org.checkerframework.checker.nullness.qual.Nullable;
 
 /**
  * This visitor removes every member in the compilation unit that is not a member of its {@link
- * #methodsToLeaveUnchanged} set or {@link #membersToEmpty} set. It also deletes the bodies of all
- * methods and replaces them with "throw new Error();" or remove the initializers of fields
- * (minimized if the field is final) within the {@link #membersToEmpty} set.
+ * #targetMethods} set or {@link #usedMembers} set. It also deletes the bodies of all methods and
+ * replaces them with "throw new Error();" or remove the initializers of fields (minimized if the
+ * field is final) within the {@link #usedMembers} set.
```

Comment:
`isUsedMethodParameterType()` documentation should also be updated to use `usedMembers` instead of `membersToEmpty`